main.py

```diff
@@ -22,3 +20,26 @@
         logger=config.get("logger", "tensorboard"),
     )
     trainer.train()
+
+
+if __name__ == "__main__":
+    parser = flags.get_parser()
+    args = parser.parse_args()
+    config = build_config(args)
+
+    if args.submit:  # Run on cluster
+        executor = submitit.AutoExecutor(folder=args.logdir)
```

Comment:
Could we make this `os.path.join(args.logdir, "%j")`?

Reply:
Done